src/App.js

```diff
@@ -1,24 +1,42 @@
+/* eslint-disable max-len */
 import React from 'react';
 import './App.css';
+import { Tabs } from './components/Tabs/Tabs';
 
-// eslint-disable-next-line no-unused-vars
 const tabs = [
   {
-    title: 'Tab 1',
-    content: 'Some text 1',
+    title: 'Home',
+    content: 'Since component logic is written in JavaScript instead of templates, you can easily pass rich data through your app and keep state out of the DOM.',
   },
   {
-    title: 'Tab 2',
-    content: 'Some text 2',
+    title: 'Profile',
+    content: 'Declarative views make your code more predictable and easier to debug.',
   },
   {
-    title: 'Tab 3',
-    content: 'Some text 3',
+    title: 'Contact',
+    content: 'React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes.',
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+class App extends React.Component {
+  state ={
+    textContent: tabs[0].content,
+  }
+
+  change = (index) => {
+    this.setState(state => ({
+      textContent: tabs[index].content,
+    }));
+  };
```

Comment:
```suggestion
  change = (index) => {
    this.setState({
      textContent: tabs[index].content,
    });
  };
```
тебе же тут не нужен prevState